src/transformState.js

```diff
@@ -5,7 +5,24 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const el of actions) {
+    switch (el.type) {
```

Comment:
Maybe it would be better not to use names like 'el' or 'element' - these names are not very semantic

Reply:
Agree